src/commons/sideContent/SideContentHtmlDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+
+export interface SideContentHtmlDisplayProps {
+  content: string;
+  handleAddHtmlConsoleError: (errorMsg: string) => void;
+}
+
+const ERROR_MESSAGE_REGEX = /^Line [\d]+: /i;
```

Comment:
```suggestion
const ERROR_MESSAGE_REGEX = /^Line \d+: /i;
```

Square brackets are unnecessary here since the character class has only one (meta)character.